src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -665,12 +668,13 @@ private IAnonymousObjectCreationOperation CreateBoundAnonymousObjectCreationExpr
             return new CSharpLazyAnonymousObjectCreationOperation(this, boundAnonymousObjectCreationExpression, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
+#nullable enable
         private IOperation CreateBoundObjectCreationExpressionOperation(BoundObjectCreationExpression boundObjectCreationExpression)
         {
             MethodSymbol constructor = boundObjectCreationExpression.Constructor;
```

Comment:
>constructor [](start = 25, length = 11)

Constructor is not nullable here, why is is marked as nullable in ObjectCreationOperation?